npm.js

```diff
@@ -59,6 +67,10 @@ exports.translate = function(load){
 				if(pkg.browser){
 					delete pkg.browser.transform;
 				}
+				if(pkg.system && pkg.system.npmAlgorithm === "nested" && context.isFlatFileStructure === true) {
```

Comment:
Honestly, I think we should remove it. I find too many warnings causes people to not read the important ones. In this case I can't think of any scenario where this would be useful...